roles/confluent.kafka_connect/library/confluent_connectors.py

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/python
+
+# Copyright: (c) 2019, Confluent Inc
+
+ANSIBLE_METADATA = {
+    'metadata_version': '0.9',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+module: confluent_connectors
+
+short_description: This module allows setting up Confluent connectors from Ansible.
+
+version_added: "2.4"
+
+description:
+    - "This module allows setting up Confluent connectors from Ansible. It registers the new ones,
+    updates the existing ones and removes the deleted ones."
+
+options:
+    connect_url:
+        description:
+            - URL of the Connect REST server to use to add/edit connectors
+        required: true
+    active_connectors:
+        description:
+            - Dict of active connectors (each connector object must have a 'name' and a 'config' field)
+        required: true
+
+author:
+    - Laurent Domenech-Cabaud (@ldom)
+'''
+
+EXAMPLES = '''
+- connect_url: {{kafka_connect_http_protocol}}://0.0.0.0:{{kafka_connect_rest_port}}/connectors
+  active_connectors: [{"name": "test-6-sink", "config": { .../... }},{"name": "test-5-sink", "config": { .../... }}]
+'''
+
+RETURN = '''
+message:
+    description: The output message that the module generates
+    type: str
+    returned: always
+'''
+
+import json
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.urls import open_url
+import ansible.module_utils.six.moves.urllib.error as urllib_error
+
+
+def get_current_connectors(connect_url):
```

Comment:
Feeling like this is almost like a DAO class you'd see in other projects `get()`, `getAll()`, `update(id,payload)`, `delete(id)`, etc. 

That being said: suggestion: `get_all`

Reply:
@cricket007 I think this makes sense as is, more descriptive.

Reply:
@cricket007 @JumaX 
After Justin's comment, I'm unclear whether you guys want me to change that name or not. In general I'm trying to make it explicit when a var holds the existing/currently running connectors and when it holds the newly defined connectors (which may include some of the existing ones of course). If we remove it, there might be confusion. Let me know...

Reply:
It seems the install one is the only one that there's a consensus on changing.

I'm honestly fine with the others. Just the naming is redundant for the Kafka Connect / connectors.py module (I'm also wondering if the file should be renamed too, since Kafka Connect API is under Apache, not Confluent licensing)

Reply:
OK no pb, I'll update the install one. 
I see what you mean regarding the file name... Do I go ahead and update everything with the new name (there's also a change to make in the tasks/main.yml), or do we want another opinion?